tsconfig.json

```diff
@@ -31,10 +31,9 @@
     "core/templates/dev/head/mathjaxConfig.ts",
     "core/templates/dev/head/expressions/ExpressionParserService.js" 
   ],
-  "include_explanation": "The files in interactions/LogicProof/static/js are yet to be integrated into webpack (#6961)",
+  "include_explanation": "The files in extensions/ckeditor_plugins are yet to be integrated into webpack.",
```

Comment:
Any issue number for this?

Reply:
No.
@NishealJ is still working with the ckeditor_plugins.

Reply:
OK. @NishealJ could you please ensure there is an issue for this? /cc @apb7

Reply:
Hi @seanlip & @Jamesjay4199,

sure I'll create an issue for it but this seems to be a ckeditor4's plugin directory please feel to add into webpack. I'm not working with this dir I have created a new dir for ck5 plugins and added it to webpack Thanks!

Reply:
Oh this is actually #6732

Reply:
OK -- @Jamesjay4199, please could you add the issue number accordingly in a TODO here. Thanks!

(And @NishealJ, no action required.)

Reply:
Done.